packages/editor-ui/src/views/TemplateView.vue

```diff
@@ -21,7 +24,12 @@
 			</div>
 			<div>
 				<div :class="$style.image">
-					<workflow-preview v-if="showPreview" :workflow="template.workflow" @close="onHidePreview" :loading="loading"/>
+					<workflow-preview
```

Comment:
nitpick - `WorkflowPreview` for our own components. easier to search for and more consistent with our editor components. let's keep snake case for our element-ui/design system components